.gitlab-ci.yml

```diff
@@ -234,32 +180,3 @@ smoke-test:
     echo "Running full suite of package tests ..."
     docker run finestructure/rester-sitemap:1.0.0 ${SITE_URL}/sitemap.xml > all-packages.restfile
     rester all-packages.restfile
-
-
-.backup-common: &backup-common
-  stage: build
-  tags:
-    - spi-prod-backup
-  script: |
-    TARFILE=spi_${ENV}_$(date +%Y-%m-%d).tgz
-    DUMPFILE=$(basename $TARFILE .tgz).dump
-    ./scripts/db_backup.sh $TARFILE $DUMPFILE
-    ./scripts/convert_to_db_dump.sh $TARFILE $DUMPFILE
-    # copy file to S3
-    S3_BACKUP_DIR=/Users/spi/Desktop/spi-db-backups
-    # mount s3fs just in case (ignoring errors when it's already mounted)
-    s3fs spi-db-backups ${S3_BACKUP_DIR} -o passwd_file=/Users/spi/.passwd-s3fs || true
-    cp $DB_BACKUP_DIR/$DUMPFILE ${S3_BACKUP_DIR}/
```

Comment:
I've removed the backup stages, because the hosted dbs are backed up automatically. We should test separately if pulling down snapshots impacts performance like it did in the docker-based env (it almost certainly won't).

Reply:
It might still be nice to have downloadable snapshots like we used to and we can bring that back if we want to after the switch over.